src/App.js

```diff
@@ -5,20 +5,23 @@ import { ProductProvider } from "./context/ProductContext/ProductContext";
 import { CartProvider } from "./context/CartContext/CartContext";
 import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
+import { WishlistProvider } from "./context/WishlistContext/WishlistContext";
 
 function App() {
   return (
     <div className="App">
       <CartProvider>
-        <ProductProvider>
-          <div className="page-layout">
-            <Header></Header>
-            <div className="page-layout__main">
-              <Router />
+        <WishlistProvider>
+          <ProductProvider>
+            <div className="page-layout">
+              <Header></Header>
+              <div className="page-layout__main m-l">
+                <Router />
+              </div>
+              <Footer></Footer>
```

Comment:
If you are not using children inside footer you can simply use self closing tag of this footer component like `<Footer />`